test/onnx/verify_onnx.cpp

```diff
@@ -242,11 +246,13 @@ TEST_CASE(batch_norm_3d_test)
     std::vector<float> result_vector;
     result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
 
+    // converted to float on visit
```

Comment:
Why does it have to converted ?

Reply:
I'm not sure, the visit returns floating point numbers with only `float32` precision even though I specified `double` precision.